play/src/front/Phaser/Game/GameManager.ts

```diff
@@ -231,6 +237,19 @@ export class GameManager {
         return this.startRoom;
     }
 
+    public enableChat() {
```

Comment:
The name "enableChat" can make you think this will actually enable the chat (and not retrieve a value).
Why not `isChatEnabled` instead?

By the way, I'm not sure those methods belong to the gameManager.
The gameManager should be somehow independant from the current room we are in.

Maybe we could push those methods in the "Room.ts" file?